EnglishTranslation-master/ETC.tsv

```diff
@@ -19640,18 +19640,18 @@ ETC_20151224_019640	Enter the amount of TP you want to gift.
 ETC_20151224_019641	You've received {Count} TP from {Giver}.
 ETC_20151224_019642	You've gifted {Count} TP to {Receiver}.
 ETC_20151224_019643	You do not have enough TP.
-ETC_20151224_019644	The person has rejected your request.
+ETC_20151224_019644	has declined your request.
```

Comment:
I mean does one need to be added? Just double checking in case it doesnt show up like
`Spadowhas declined your request`?